src/lib/tray.py

```diff
@@ -221,9 +224,14 @@ def update_dropdown_menus(self):
             self.menu_count.setText(_("{updates} updates available").format(updates=updates_count))
             self.menu_count.setEnabled(True)
 
+        # Update last check timestamp (always False to not pull unwanted attention)
+        self.menu_last_check.setText(_("Last check:\n{time}").format(time=last_check_time))
+        self.menu_last_check.setEnabled(False)
+
         # Clear the menu (to update entries)
         self.menu.clear()
         self.menu.addAction(self.menu_count)
+        self.menu.addAction(self.menu_last_check)
```

Comment:
That's purely aesthetic but I think I'd rather move this under the update section counts when there are update available (Packages, AUR, Flatpak), so at line`282` I think?

Reply:
Yes, still above the separator

Reply:
Will you push the related change or should I?